capa/features/extractors/pefile.py

```diff
@@ -87,8 +88,8 @@ def extract_file_section_names(pe, **kwargs):
         yield Section(name), AbsoluteVirtualAddress(base_address + section.VirtualAddress)
 
 
-def extract_file_strings(buf, **kwargs):
```

Comment:
weren't the kwargs meant so that we could pass `{"min_len": ...}` as such here and don't have to change all other signatures?

Reply:
I thought that using ctx was so that kwargs doesn't have to be added to all the function signatures.
But I can change it back to use kwargs for this function if it is better.